keyboards/splitkb/kyria/keymaps/john-ezra/keymap.c

```diff
@@ -167,6 +219,14 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
         }
       }
       break;
+    case MSS_CTL:
+      if (record->tap.count && record->event.pressed) {
+          tap_code16(C(KC_UP));
+          return false;
+      } else if (record->event.pressed) {
+          return true;
+      }
```

Comment:
```suggestion
      if (record->event.pressed &&record->tap.count) {
          tap_code16(C(KC_UP));
          return false;
      }
```
Can be simplified a little.